src/readbin/func_start.mli

```diff
@@ -0,0 +1,10 @@
+open Bap.Std
+open Core_kernel.Std
+open Cmdliner
+
+(** [of_truth truth ~testbin] given the test binary testbin, returns
+    the function start address from ground truth truth *)
+val of_truth: string -> testbin:string -> Addr.Set.t Or_error.t
```

Comment:
it looks nicer when you delimit parameters in a doc string with square brackets, e.g.,
```
(** [of_truth truth ~testbin] given the test binary [testbin], returns
   the function start address from ground truth [truth] *)
```
In that case they will be printed in different font and will be easily detected by an API reader as formals, not as part of informal text.